tbtc-ts/test/data/sweep.ts

```diff
@@ -207,3 +214,186 @@ export const sweepWithMainUtxo: SweepTestData = {
     },
   },
 }
+
+/**
+ * Represents data for tests of assembling sweep transactions.
+ */
+export interface SweepProofTestData {
+  clientData: {
```

Comment:
What's the reasoning behind naming it `clientData`? What is the "client" here?

Reply:
It's data that that needs to be returned from the bitcoin client mock, as opposed to the expected data, or data that drives the test (arguments).

Reply:
Ok got it! So maybe we can avoid ambiguity by naming it, for example, `bitcoinChainData`?

Reply:
Done.